account/repositories/accountRepository.js

```diff
@@ -1,15 +1,22 @@
 import { MongoClient } from "mongodb";
+import * as dotenv from 'dotenv'
```

Comment:
the [documentation](https://github.com/motdotla/dotenv#usage) says: As early as possible in your application, import and configure dotenv

but i don't think you did